src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/IHtmlFragmentCache.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// An implementation of this interface provides a service to cache html fragments.
+    /// One client of this service is <see cref="CacheTagHelper"/>.
+    /// </summary>
+    public interface IHtmlFragmentCache
+    {
+        /// <summary>
+        /// Evaluates the <see cref="IHtmlContent"/> to render and caches it.
```

Comment:
Don't think so. `Evaluates` and `caches` have the same subject in this case. Changing it would give it another meaning, which you might find better, but let's ask @dougbu.